lib/AST/ASTContext.cpp

```diff
@@ -3951,8 +3951,17 @@ bool ASTContext::isStandardLibraryTypeBridgedInFoundation(
           nominal == getStringDecl() ||
           nominal == getErrorDecl() ||
           nominal == getAnyHashableDecl() ||
-          // Weird one-off case where CGFloat is bridged to NSNumber.
-          nominal->getName() == Id_CGFloat);
+          // Foundation is a dependency of CoreGraphics, but
```

Comment:
@atrick It looks like we *do* try to make optimization assumptions for the lack of `_ObjectiveCBridgeable` conformances by disallowing conformances to that protocol outside of the type's defining module, except for this hardcoded list of exceptions. I'm not sure how valuable that is anymore in the world of `id`-as-`Any`, since there's at least one way to bridge everything to and from a class type now.